### ISSUE TITLE
Error in --stream documentation example

### ISSUE THREAD

**Issue by djerius** [jqlang/jq#2095]
**Describe the bug**

The documentation for --stream includes an example:

> For example, `"a"` becomes `[[],"a"]`, and `[[],"a",["b"]]`
> becomes `[[0],[]]`, `[[1],"a"]`, and `[[1,0],"b"]`.

However, the specified output is incorrect.

**To Reproduce**
I used `jq` to test the example:

```
% echo '[[],"a",["b"]]' |  jq  -c --stream
[[0],[]]
[[1],"a"]
[[2,0],"b"]
[[2,0]]
[[2]]
```

**Expected behavior**

I expected to see `jq`'s output of  `[[2,0],"b"]` rather than the documented `[[1,0],"b"]`. 

**Environment (please complete the following information):**
 - OS and Version: Linux, Debian 10
 - jq version: Self compiled 1.6

**Comment by itchyny** [comment]
Duplication of #1279, #2010.

**Comment by itchyny** [comment]
Resolved by #2387, so closing.
